dhctl/pkg/operations/bootstrap.go

```diff
@@ -292,18 +292,21 @@ func ConnectToKubernetesAPI(sshClient *ssh.Client) (*client.KubernetesClient, er
 }
 
 const rebootExitCode = 255
+const alternativeRebootExitCode = 1
```

Comment:
We speak about rebooting master node is not the preffered way to solve problem with disks. So why we add this code ?

Reply:
This solves another problem. In Altlinux, the reboot command returns a non-standard code, which causes dhtl to crash.

Reply:
need to comment this case at least